src/common/__tests__/dateHelpers.test.js

```diff
@@ -53,4 +53,39 @@ describe('DateHelpers', () => {
     expect(getRangedMonthDateTime('april')).toMatchSnapshot('get a specific month by name');
     expect(getRangedMonthDateTime(3)).toMatchSnapshot('get a specific month by number');
   });
+
+  it('setEndOfDay should change date to end of day', () => {
```

Comment:
consistency, lets update the `it should` copy to be consistent for all of the new tests
```
it('should setEndOfDay date to...
it('should setStartOfDay date to...
etc 
```

Reply:
updated so they all start with 'should...'